doc/README.md

```diff
@@ -161,9 +175,9 @@ return status
 ```
 For each request ARC receives in standard format, it must request the utxos from the Bitcoin node. This is a slow process, as it requires a round trip to the Bitcoin node for each input.
 
-For this reason it is expected that transactions come in an extended format. Transactions in standard format requires a pretreatment to convert each tx into one in extended format to pass through the ARC pipeline.
+For this reason it is expected that transactions come in an extended format or BEEF. Transactions in standard format requires a pretreatment to convert each tx into one in extended format to pass through the ARC pipeline.
 
-That pretreatment detracts from the expected efficiency of the process because it requires extra requests to the Bitcoin network. Therefore, it is expected that the trend will be to use the extended format, which may be the only one supported in the future.
+That pretreatment detracts from the expected efficiency of the process because it requires extra requests to the Bitcoin network. Therefore, it is expected that the trend will be to use the extended format or BEEF, which may be the only ones supported in the future.
```

Comment:
```suggestion
That pretreatment detracts from the expected efficiency of the process because it requires extra requests to the Bitcoin network. Therefore, it is expected that the trend will be to use the EF or BEEF, which may be the only supported formats in the future.
```